elements/lisk-api-client/src/api_client.ts

```diff
@@ -51,7 +51,7 @@ export class APIClient {
 		return this._channel.invoke(actionName, params);
 	}
 
-	public subscribe<T = Record<string, unknown>>(eventName: string, cb: EventCallback<T>): void {
+	public subscribe(eventName: string, cb: EventCallback): void {
```

Comment:
Should not we make this one generic as we did for the other clients? 
```suggestion
	public subscribe<T = Record<string, unknown>>(eventName: string, cb: EventCallback<T>): void {
```

Reply:
i thought i updated, maybe it was out of commit...